CommonClient.py

```diff
@@ -243,7 +244,7 @@ async def disconnect(self):
         if self.server_task is not None:
             await self.server_task
 
-    async def send_msgs(self, msgs):
+    async def send_msgs(self, msgs: typing.List[typing.Dict[str, typing.Any]]) -> None:
```

Comment:
I'm not sure if you're suggesting a change here.

Guido van Rossum said he uses `Dict[str, Any]` when it works or `Any` when it doesn't. ( https://github.com/python/typing/issues/182 )

So that would be `msgs: typing.List[typing.Any]` for us here.

If we wan't to be more specific, we could do something like:

```
MessageType = typing.Union[
    typing.Dict[str, typing.Any],
    typing.Tuple[typing.Any, ...],
    typing.List[typing.Any],
    typing.Set[typing.Any],
]

async def send_msgs(self, msgs: typing.List[MessageType]) -> None:
```

Reply:
i think `msgs: typing.List[typing.Any]` is what i would expect there